dags.yaml

```diff
@@ -287,3 +287,12 @@ bqetl_marketing_fetch:
     retries: 2
     retry_delay: 30m
 
+bqetl_desktop_platform:
+  schedule_interval: 0 3 * * *
+  default_args:
+    owner: jklukas@mozilla.com
+    start_date: "2020-12-10"
+    email: ["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"]
```

Comment:
It looks like you have your email listed in the DAG itself; that should be reflected in this yaml.